torchrl/_utils.py

```diff
@@ -228,18 +240,52 @@ class implement_for:
 
     # Stores pointers to fitting implementations: dict[func_name] = func_pointer
     _implementations = {}
+    _setters = []
 
     def __init__(
-        self, module_name: str, from_version: str = None, to_version: str = None
+        self,
+        module_name: Union[str, Callable],
+        from_version: str = None,
+        to_version: str = None,
     ):
         self.module_name = module_name
         self.from_version = from_version
         self.to_version = to_version
+        implement_for._setters.append(self)
+
+    @staticmethod
+    def check_version(version, from_version, to_version):
```

Comment:
FYI, the previous implementation of check_version had a bug:
`check_version("0.9.0", "0.1.0", "0.21.0")` returned `False`